tests/static-check/run_checks.sh

```diff
@@ -5,25 +5,25 @@ set -x
 n_procs="$(getconf _NPROCESSORS_ONLN)"
 
 function check_with_gcc() {
-  rm -f config.cache
   make clean
-  ./configure -C --enable-debug CC=gcc
+  ./configure --enable-debug CC=gcc
   local gcc_exceptions="-Wno-sign-compare -Wno-enum-int-mismatch"
   make -j -l${n_procs} --keep-going CFLAGS="-Werror -Wall -Wextra $gcc_exceptions"
 }
 
 function check_with_clang() {
-  rm -f config.cache
   make clean
-  ./configure -C --enable-debug CC=clang
+  ./configure --enable-debug CC=clang
```

Comment:
I have added more about why this change in the commit comment but don't want to waste GH job resources on that just yet.

Reply:
Curious to see that because `-C` after `config.cache` should only ensure the checks for *core* are reused for *libntech*, AFAICT.

Reply:
ah, right, I didn't think of libntech benefiting from core, does it? or would the cache be separate since it is a subdir? I will test actually. :+1: